### PR TITLE
refactor: vlad points

### PR DESCRIPTION
**Pull request by Vlad-2109**
All points from conventions list are fixed)

### CODE REVIEW

File: src/components/dashboard/coursesList/coursesList.tsx
Comment (Nazaziik) [inline src/components/dashboard/coursesList/coursesList.tsx:1]: Should the name of component be in PascalCase ?

File: src/components/dashboard/enrolledCardOfCourse/EnrolledCardOfCourse.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import { Box, CardMedia, Typography } from '@mui/material';
+import { EnrolledCoursesData } from '../../../types/components/componentType';
+import './_enrolledCard.scss';
```
Comment (Nazaziik) [inline src/components/dashboard/enrolledCardOfCourse/EnrolledCardOfCourse.tsx:5]: import SCSS file like module component ?

File: src/components/calendarComponent/calendarComponent.tsx [hunk 2]
```diff
@@ -3,16 +3,16 @@ import Calendar from 'react-calendar';
 import 'react-calendar/dist/Calendar.css';
 import './calendar.scss';
```
Comment (Nazaziik) [inline src/components/calendarComponent/calendarComponent.tsx:4]: import SCSS file like module component ?

File: src/components/dashboard/courseProgress/CourseProgress.tsx
[the remark this answers is not in the document]
Comment (Vlad-2109) [reply src/components/dashboard/courseProgress/CourseProgress.tsx:1]: It's a feature of JEST testing.

File: src/components/dashboard/coursesList/coursesList.tsx
Comment (Vlad-2109) [reply src/components/dashboard/coursesList/coursesList.tsx:1]: It's a feature of JEST testing.

File: src/components/dashboard/enrolledCardOfCourse/EnrolledCardOfCourse.tsx (hunk 1 above)
Comment (Vlad-2109) [reply src/components/dashboard/enrolledCardOfCourse/EnrolledCardOfCourse.tsx:5]: Classnames are not used in tsx

File: src/components/calendarComponent/calendarComponent.tsx (hunk 2 above)
Comment (Vlad-2109) [reply src/components/calendarComponent/calendarComponent.tsx:4]: Calendar has its own styles, which has conflicts with modules styles.

**Review by Virsta — APPROVED** [review]

**Review by Nazaziik — APPROVED** [review]

**Pull request merged**